modules/administration-guide/partials/proc_installing-image-puller-using-che-operator.adoc

```diff
@@ -0,0 +1,87 @@
+[id="installing-image-puller-using-che-operator_{context}"]
+= Installing {image-puller-name-short} using the {prod-short} Operator
+
+This section describes how to use the {prod-short} Operator to install the {image-puller-name-short}. This is a Community-Supported tech preview feature.
+
+.Prerequisites
+
+* xref:defining-the-list-of-images-to-pull.adoc[]
+* xref:defining-the-memory-parameters-for-the-image-puller.adoc[]
+ifeval::["{project-context}" == "che"]
+* Operator Lifecycle Manager is available on the {kubernetes} instance.
+endif::[]
+* OperatorHub is available on the OpenShift instance.
+* The {prod-short} Operator is available. See xref:installation-guide:installing-che-on-openshift-4-using-operatorhub.adoc[]
+
+.Procedure
+
+. Edit the `CheCluster` Custom Resource and set `.spec.imagePuller.enable` to `true`
++
+.Enabling {image-puller-name-short} in the `CheCluster` Custom Resource 
+====
+[source,yaml,subs="+attributes"]
+----
+apiVersion: org.eclipse.che/v1
+kind: CheCluster
+metadata:
+  name: {prod-checluster}
+spec:
+  # ...
+  imagePuller:
+    enable: true
+----
+====
++
+[NOTE]
+.Uninstalling {image-puller-name-short} using {prod-short} Operator
+====
+* Edit the `CheCluster` Custom Resource and set `.spec.imagePuller.enable` to `false`:
+====
+
+. Edit the `CheCluster` Custom Resource and set the `.spec.imagePuller.spec` to configure the {image-puller-name-short} parameters for the {prod-short} Operator.
++
+.Configuring {image-puller-name-short} in the `CheCluster` Custom Resource 
+====
+[source,yaml,subs="+quotes,+attributes,+macros"]
+----
+apiVersion: org.eclipse.che/v1
+kind: CheCluster
+metadata:
+  name: {prod-checluster}
+spec:
+  ...
+  imagePuller:
+    enable: true
+    spec:
+        configMapName: __<{image-puller-deployment-name}>__
+        daemonsetName: __<{image-puller-deployment-name}>__
+        deploymentName: __<{image-puller-deployment-name}>__
+        images: __<{image-puller-images}>__
```

Comment:
`images` is a comma-separated list? It's not clear.

Reply:
cc @ibuziuk

Reply:
yes, it was part of the part of original PR - https://github.com/eclipse/che-docs/pull/1800/files#diff-3c2acb4fad07efec5d4153711382a72724f98158b4086d6686b1c7ce461c6b96R80

@themr0c could you please backport it properly. Frankly, I like the original PR more - not sure why you opted for redoing it

Reply:
The example is using the attribute, that we already have defined there: https://github.com/eclipse/che-docs/blob/master/antora-playbook.yml#L55 -- Values in upstream and downstream are different, using an attribute is one way to modify the content depending on the context.

New PR: because @tomgeorge left Red Hat, he wouldn't merge the suggestions that I left as a PR in his fork. That was the only way to get the PR out of the attic.

Reply:
Is it acceptable like that?

![Screenshot from 2021-03-02 12-54-14](https://user-images.githubusercontent.com/243761/109644938-6eb9b400-7b56-11eb-88b0-de27ae000afa.png)

Reply:
The list of `images` separated by a semicolon, should not have been put in `<>` since we are using the exact names of the real images. `<>` is for values that should be defined by a user, such as a workspace name __<Myworkspace>__ .
@themr0c